src/runner.ts

```diff
@@ -133,7 +133,7 @@ export async function run(fn: Runner, args: string[], options: DetectOptions = {
     console.log(`ni    ${dash}  install`)
     console.log(`nr    ${dash}  run`)
     console.log(`nlx   ${dash}  execute`)
-    console.log(`nu    ${dash}  upgrade`)
+    console.log(`nup    ${dash}  upgrade`)
```

Comment:
I forgot to remove a space 😭

Reply:
@antfu can you remove that extra space instead of me opening another pr for that 😭

Reply:
is failing?